src/v/kafka/server/handlers/describe_acls.cc

```diff
@@ -84,6 +84,20 @@ ss::future<response_ptr> describe_acls_handler::handle(
         co_return co_await ctx.respond(std::move(resp));
     }
 
+    /// To prevent stale responses in the case this node is not the controller
+    /// leader, wait until it catches up with the current controller leaders
+    /// current last_applied offset of the controller log
+    static const auto catchup_timeout = 5s;
+    const auto error_code = co_await ctx.security_frontend()
+                              .wait_until_caughtup_with_leader(catchup_timeout);
+    if (error_code) {
+        vlog(
+          klog.info,
```

Comment:
Should this be a warning?

Reply:
I didn't think it warranted that since this is still primarily a best effort attempt to grab the freshest data

Reply:
Sounds good.